backend/server.js

```diff
@@ -151,49 +151,32 @@ app.get("/scoutOpponent", async (req, res) => {
   return {};
 });
 
-//POSTS
-app.post("/player", async (req, res) => {
+//UPDATES
+app.patch("/player", async (req, res) => {
   const playerType = req.body.playerType;
   const playerId = parseInt(req.body.playerId);
   const userId = parseInt(req.body.userId);
-  const playerName = decodeURI(req.body.playerName);
 
   try {
-    const existingPlayer = await prisma[playerType].findMany({
+    const updateUser = await prisma.user.update({
       where: {
-        playerId,
-        userId,
+        id: userId,
       },
-    });
-    if (existingPlayer.length != 0) {
-      return res
-        .status(400)
-        .json({ error: "Player has already been added to MyTeam" });
-    }
-
-    const performanceScores = await calcPerformanceScores(playerId);
-
-    const newPlayer = await prisma[playerType].create({
       data: {
-        playerId,
-        userId,
-        playerName,
-        outsideOffenseScore: performanceScores.outsideOffenseScore,
-        insideOffenseScore: performanceScores.insideOffenseScore,
-        offenseDisciplineScore: performanceScores.offenseDisciplineScore,
-        defenseDisciplineScore: performanceScores.defenseDisciplineScore,
-        consistencyScore: performanceScores.consistencyScore,
-        reboundingScore: performanceScores.reboundingScore,
+        [playerType]: {
+          connect: {
+            id: playerId,
+          },
```

Comment:
<img width="1728" alt="Screenshot 2024-07-18 at 11 46 19 AM" src="https://github.com/user-attachments/assets/54644d8f-e371-4a40-99fa-2a91116fdd19">
Instead of creating a whole new entry, link the existing Player